examples/delete_disks.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Show how to delete many disks created by the 'add_disks.py' example quickly.
+"""
+
+import time
+from contextlib import closing
+
+from ovirtsdk4 import types
+
+from helpers import common
+from helpers.common import progress
+from helpers.units import humansize, GiB
+
+parser = common.ArgumentParser(description="Delete disks created by the 'add_disks.py' example")
+
+parser.add_argument(
+    "--sd-name",
+    required=True,
+    help="Name of the storage domain.")
+
+parser.add_argument(
+    "--description",
+    default="Created by add_disks.py",
+    help="The description of the disks to be deleted.")
+
+args = parser.parse_args()
+
+progress("Connecting...")
+connection = common.create_connection(args)
+with closing(connection):
+    disks_service = connection.system_service().disks_service()
+    storage_domains_service = connection.system_service().storage_domains_service()
+    sd = storage_domains_service.list(search='name={}'.format(args.sd_name))[0]
```

Comment:
This can raise IndexError:

    storage_domains_service.list(search='name={}'.format(args.sd_name))[0]

If the storage domain does not exist, meaning the user made an error. It would
be nicer to do:

    found = storage_domains_service.list(search=f'name={args.sd_name}')
    if not found:
        raise RuntimeError(f"Cannot find storage domain {args.sd_name}")

    sd = found[0]

Reply:
ack